x-pack/plugins/task_manager/server/lib/calculate_health_status.ts

```diff
@@ -16,12 +16,12 @@ export function calculateHealthStatus(
   config: TaskManagerConfig,
   shouldRunTasks: boolean,
   logger: Logger
-): { status: HealthStatus; reason?: string } {
+): { status: HealthStatus; reason?: string; isEmpty?: boolean } {
```

Comment:
maybe instead of `isEmpty`, we could just have a new `HealthStatus.NOT_INITIALIZED` and filter on that instead?

Reply:
That sounds like a great idea!